vm/interpreter/src/gas_block_store.rs

```diff
@@ -9,6 +9,7 @@ use std::cell::RefCell;
 use std::rc::Rc;
 
 /// Blockstore wrapper to charge gas on reads and writes
+#[derive(Clone)]
```

Comment:
Do we need this?

Reply:
We do. That became needed as soon as we made `BlockStore: Clone`